routers/web/org/home.go

```diff
@@ -164,5 +152,7 @@ func Home(ctx *context.Context) {
 	ctx.Data["Page"] = pager
 	ctx.Data["ContextUser"] = ctx.ContextUser
 
+	ctx.Data["ShowMemberTeamBoard"] = ctx.Org.IsMember || len(members) > 0
```

Comment:
Shouldn't that be `ShowMemberAndTeamTab`?

Reply:
Done.